prow/plugins/slackevents/slackevents.go

```diff
@@ -98,7 +98,7 @@ func notifyOnSlackIfManualMerge(pc client, pe github.PushEvent) error {
 	if mw := getMergeWarning(pc.SlackConfig.MergeWarnings, pe.Repo.Owner.Login, pe.Repo.Name); mw != nil {
 		//If the MergeWarning whitelist has the merge user then no need to send a message.
 		if !stringInArray(pe.Pusher.Name, mw.WhiteList) && !stringInArray(pe.Sender.Login, mw.WhiteList) {
-			message := fmt.Sprintf("Warning: <@%s> manually merged %s", pe.Sender.Login, pe.Compare)
+			message := fmt.Sprintf("*Warning:* <@%s> manually merged %s", pe.Sender.Login, pe.Compare)
```

Comment:
Will the angle brackets in `<@%s>` render now that username linking is disabled?

Reply:
I don't think there is a way via the api for it to be opportunistic unless we list the username twice (once without the @ and once with). If the brackets weren't rendering before, then perhaps they just need to be escaped.

Reply:
No, I'm saying I don't think we want them to render.
Based on [this](https://api.slack.com/docs/message-formatting#how_to_escape_characters) I don't think they will render. We might want to remove them because they don't do anything, but it shouldn't make a difference.